c7n/cli.py

```diff
@@ -401,3 +401,7 @@ def main():
             raise
         traceback.print_exc()
         pdb.post_mortem(sys.exc_info()[-1])
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
This was added for convenience - it can help test changes by directly running the python file.